governance-exercises/JanSky Polling Exercise 4 - Forum Post.md

```diff
@@ -0,0 +1,20 @@
+# Forum Response
+
+Author: JanSky  
+Date: Wednesday 9 August 2023
+
+---
+
+This situation requires an immediate response. A couple of recommendations that should happen promptly:
+
+**Emergency Poll on Fix A**: I will be initiating an emergency poll presently to see if there is community support for Fix A. **This poll will only last 12 hours, @Aligned Delegates please vote as soon as you are ready and able.** While [MIP24](https://mips.makerdao.com/mips/details/MIP24#MIP24c3) states that a poll might not be strictly necessary in emergencies, I am recommending it given the difference of opinion between Roundtable and WeCodeCrypto.
+
+**Emergency Governance Call**: I am organizing a governance call with Roundtable Develop, WeCodeCrypto, and Imogen. @Ecosystem Facilitator should also attend if they are able (please message me on Discord for information, you have a message there in your inbox). I will summarize and release notes from this call to the community once this Emergency Response stops.
+
+
+Imogen's expertise in Executive spell preparation should be respected. As Protocol Scope Facilitator, it is their call what goes in the emergency executive. I hope the poll and emergency call coordination aid their decision, but please note Imogen is not bound by the results. Should large support for Fix A arise during the emergency poll, I will advise initiating preparations for its implementation before the polling period ends.
```

Comment:
Phrasing is a bit screwy here as well. Basically, despite Henrik mouthing off, Imogen appears entirely in control of the situation.

"Imogen's expertise in Executive spell preparation should be respected." This implies that someone is disrespecting it, and that Imogen is incapable of dealing with someone disrespecting their role. Neither of those things are the case, which means you even bringing it up is weird, because it implies that you think it needs to be clarified, and thus that you harbour doubts about Imogen's expertise.

Reply:
"will advise initiating preparations for its implementation before the polling period ends." - Roundtable has already said they are doing this.